src/tui.rs

```diff
@@ -393,7 +398,8 @@ impl ToipeTui {
 
                 // clear line
                 line = Vec::new();
-                current_len = 0;
+                line.push(word.clone());
```

Comment:
you can have it in the vec while creating it.
```rust
line = vec![word.clone()];
```

Reply:
done